Doc/library/collections.abc.rst

```diff
@@ -213,6 +213,9 @@ Collections Abstract Base Classes -- Detailed Descriptions
 
    ABC for classes that provide the :meth:`~object.__call__` method.
 
+   See :ref:`annotating-callables` for details on how to use
+   :class:`Callable` and :class:`~typing.Callable` in type annotations.
```

Comment:
Probably don't need to mention the deprecated alias if we're already in the docs for the non-deprecated object. And let's get rid of the link that only takes us to `collections.abc.Callable`, which is where we already are ;)

```suggestion
   :class:`!Callable` in type annotations.
```

Reply:
I think this bit is relevant enough here - a *lot* of code uses deprecated `typing` aliases, and having both listed here as equivalent helps to understand relationship of these aliases.

Reply:
Fair enough! Suggestion withdrawn

Reply:
Sorry, no, agreed - that isn't necessary in `collections.abc` notes. Mentioning both in `typing.rst` guidelines section is sufficient.